FLEDGE.md

```diff
@@ -121,6 +122,24 @@ The `adComponents` field contains the various ad components (or "products") that
 The browser will provide protection against microtargeting, by only rendering an ad if the same rendering URL is being shown to a sufficiently large number of people (e.g. at least 100 people would have seen the ad, if it were allowed to show).  As discussed in the [Outcome-Based TURTLEDOVE](https://github.com/WICG/turtledove/blob/master/OUTCOME_BASED.md) proposal, this threshold applies only to the rendered creative; all of the metadata associated with ad bidding is not under any such restriction.  Since a single interest group can carry multiple possible ads that it might show, the group will have an opportunity to re-bid another one of its ads to act as a "fallback ad" any time its most-preferred choice is below threshold.  This means that a small, specialized interest group that is still below the daily-update threshold could still choose to participate in auctions, bidding with a more-generic ad until the group becomes large enough.
 
 
+#### 1.3 Permission Delegation
```

Comment:
Unfortunately, I don't know how to add such a link, and there are no examples of a link to a section in this document (Also not sure how to compile markdown locally, or if it's possible to view the fully formatted version on github)

Reply:
The markdown for the link would be
`(see [1.3 Permission Delegation](13-permission-delegation))`

Reply:
Thanks, Michael - I really should have been able to figure that one out.

Reply:
> I don't understand why per-frame deeply changes things — checking for permission can still block on a network request, so the return status should be async-friendly. I think the promise is a sound design in any case.

While per-frame vs per-page does matter from a cross-site spying standpoint (at least if we use two-value NIKs / network partition keys instead of one value ones), the actual distinction I meant to focus on was:

1) A permanent (per page load) join/leave limit (pages can only join 100 cross-origin IGs, which we allow to all queue at any time, and (possibly) will continue to run in the background on navigation away, modulo some sort of global limit.  This approach does not need promises (which is something Paul is skeptical of)
2) A simultaneous join/leave limit, with quote opening back up once we've retrieved .well-known files.  This approach needs promises information of join/leave completion.